tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Models/Matter/MatterDetails.cs

```diff
@@ -43,6 +43,12 @@ public string SubareaOfLaw
             set;
         }
 
+        public IDictionary<string, ManagedColumn> ManagedColumnTerms

```

Comment:
To support dynamic taxonomy hierarchy, this new property has been added. ManagedColumn is a C# model class, which consist of three properties:
TermName
Id 
WssId